app/controllers/doctors_controller.rb

```diff
@@ -30,28 +31,34 @@ def edit
 end
 
 def update
+  @doctor = Doctor.find(params[:id])
+  @doctor.update(doctor_params)
```

Comment:
Update automatically saves to database (like .create vs .new), if we change to [one of these other methods](http://www.davidverhasselt.com/set-attributes-in-activerecord/), we can write in an 
`if @doctor.save`
   `redirect_to doctor_path`
`else`
  `flash[:notice] = "Error etc etc etc"`
   `redirect_to new_doctor_path`

This will more elegantly handle errors in saving